doc/source/examples/unresolved-cfd-dem/gas-solid-spouted-cylinder-bed/gas-solid-spouted-cylinder-bed.rst

```diff
@@ -0,0 +1,300 @@
+
+.. code-block:: text
+
+    subsection lagrangian physical properties
+      set gx                       = 0
+      set gy                       = 0
+      set gz                       = -9.81
+      set number of particle types = 1
+      subsection particle type 0
+        set size distribution type            = uniform
+        set diameter                          = 0.0025
+        set number                            = 100000
+        set density particles                 = 500
+        set young modulus particles           = 10000000
+        set poisson ratio particles           = 0.25
+        set restitution coefficient particles = 0.97
+        set friction coefficient particles    = 0.4
+        set rolling friction particles        = 0.3
+      end
+      set young modulus wall           = 10000000
+      set poisson ratio wall           = 0.25
+      set restitution coefficient wall = 0.33
+      set friction coefficient wall    = 0.2
+      set rolling friction wall        = 0.3
+    end
+
+Insertion Info
+~~~~~~~~~~~~~~~~~~~
+
+The ``insertion info`` subsection manages the insertion of particles. This section has to be modified because of the change in particle number and use of a different geometry. The insertion box parameter was set as follows so that it could fit in the cylinder.
+
+.. code-block:: text
+
+    subsection insertion info
+      set insertion method                               = volume
+      set inserted number of particles at each time step = 100000
+      set insertion frequency                            = 2000
+      set insertion box minimum x                        = -0.075
+      set insertion box minimum y                        = -0.075
+      set insertion box minimum z                        = 0
+      set insertion box maximum x                        = 0.075
+      set insertion box maximum y                        = 0.075
+      set insertion box maximum z                        = 0.07
+      set insertion distance threshold                   = 1.05
+      set insertion maximum offset                       = 0.3
+      set insertion prn seed                             = 19
+    end
+
+Floating Walls
+~~~~~~~~~~~~~~~~~~~
+
+When we pack the cylinder with particles, we need to keep them inside and prevent them from falling through the small inlet channel. To do so, we place the stopper at the bottom of the cylinder, which is :math:`z = 0` plain, as in:
+
+.. code-block:: text
+
+    subsection floating walls
+      set number of floating walls = 1
+      subsection wall 0
+        subsection point on wall
+          set x = 0
+          set y = 0
+          set z = 0
+        end
+        subsection normal vector
+          set nx = 0
+          set ny = 0
+          set nz = 1
+        end
+        set start time = 0
+        set end time   = 50
```

Comment:
Why a end time of 50? Ideally the end time should be very large no? to ensure itnever disappears

Reply:
Exactly. I changed into 999 s